samples/dnn/facial_landmarks_ie.cpp

```diff
@@ -0,0 +1,195 @@
+// This file is part of OpenCV project.
```

Comment:
Sorry for confuse but license header is not need in the samples folder

Reply:
The strings 1-7 should be removed?

Reply:
There is the license header in, for example, colorization sample..